packages/adapter-neon/src/conversion.ts

```diff
@@ -38,6 +38,132 @@ const ArrayColumnType = {
   XML_ARRAY: 143,
 }
 
+export class UnsupportedNativeDataType extends Error {
```

Comment:
This information was in the rust driver internals. We need it here to make sense of "Unsupported column type $NUMBER"